src/store/index.js

```diff
@@ -1,6 +1,11 @@
-import { createStore } from 'redux';
+import { createStore, applyMiddleware } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import thunk from 'redux-thunk';
 import rootReducer from '../reducers/index';
```

Comment:
Novamente a sugestão do espaço entre as importações, mas lembrando que é somente uma sugestão para uma boa prática!
```suggestion

import rootReducer from '../reducers/index';
```